src/Blazored.Modal/BlazoredModal.razor.cs

```diff
@@ -14,19 +14,21 @@ public class BlazoredModalBase : ComponentBase, IDisposable
         protected bool IsVisible { get; set; }
         protected string Title { get; set; }
         protected RenderFragment Content { get; set; }
-        protected ModalParameters Parameters { get; set; }
+        protected ModalParameters Parameters { get; set; }
+        protected string Style { get; set; }
 
         protected override void OnInitialized()
         {
             ((ModalService)ModalService).OnShow += ShowModal;
             ModalService.OnClose += CloseModal;
         }
 
-        public void ShowModal(string title, RenderFragment content, ModalParameters parameters)
+        public void ShowModal(string title, RenderFragment content, ModalParameters parameters, string style)
         {
             Title = title;
             Content = content;
             Parameters = parameters;
+            Style = style ?? "blazored-modal";
```

Comment:
This may not be needed anymore - See comment above.

Another option would be to leave the `@Style` usage in the `BlazoredModal` component and change the code here to something like this.

```suggestion
            Style = string.IsNullOrWhitespace(Style) ? "blazored-modal" : $"blazored-model {Style}";
```